src/bench/bench.h

```diff
@@ -14,6 +14,9 @@
 #include <boost/preprocessor/cat.hpp>
 #include <boost/preprocessor/stringize.hpp>
 
+struct RegTestingSetup;
```

Comment:
Maybe I oversaw it but is it really necessary to include this variable into the header?
As far as I know it should work just fine if just the pointer is included in the header file.

Reply:
This is not declaring a variable, it's forward declaring the type RegTestingSetup.

Reply:
But is it still required?

Reply:
Yes, it's defined in src/test/util/setup_common.h which isn't included at this point.